src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,117 @@
+package frc.team2412.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    // Constants
+
+    public static class IntakeConstants {
+        public static String teamColor = "placeholder";
+        public static final double INTAKE_SPEED = 1;
+
+    }
+
+    // Enums
+
+    public enum IntakeMotorState {
+        IN, OUT, STOPPED;
+    }
+
+    public enum IntakeSolenoidState{
+        EXTENDED, RETRACTED;
+    }
+    
+    
+    public enum IntakeState{
+        EXTEND(DoubleSolenoid.Value.kForward),
+        RETRACT(DoubleSolenoid.Value.kReverse);
+
+        public DoubleSolenoid.Value value;
+
+        private IntakeState(DoubleSolenoid.Value value) {
+            this.value = value;
+        }
+    }
+    
+
+
+    // Define Hardware
+
+    private final WPI_TalonFX motor1;
+
+    private final WPI_TalonFX motor2;
+
+    private final DoubleSolenoid solenoid1;
+
+    //private final DoubleSolenoid solenoid2;
+
```

Comment:
If the current CAD only has a single control solenoid, then just delete this instead of leaving it commented out.  It's trivial to add it back later if needed.